src/main.rs

```diff
@@ -15,6 +15,22 @@ use std::time::Duration;
 
 use directories_next::ProjectDirs;
 
+#[cfg(feature = "systray")]
+mod tray_icon;
+
+#[cfg(not(feature = "systray"))]
```

Comment:
I think I would actually remove this "fake" module, and instead add `#[cfg(feature = "systray")]` to each of the methods that call into this below. Makes it a little clearer what's going on when you read that code directly.